package.json

```diff
@@ -3,8 +3,8 @@
 	"version": "0.2.0",
 	"description": "Cookie Analysis Tool and CLI for analysis and understanding of cookie usage on web pages.",
 	"scripts": {
-		"cli:dev": "npm run dev --workspace=@cookie-analysis-tool/cli",
-		"cli:build": "npm run build --workspace=@cookie-analysis-tool/cli",
+		"cli:dev": "tsc-watch",
+		"cli:build": "tsc --build",
```

Comment:
Yes, TSC is still being used to build CLI but the npm script is moved to its package. In future if any changes are to be done to its build routine it will be done in its workspace.